webpack.config.js

```diff
@@ -57,7 +57,7 @@ module.exports = (env, argv) => ({
     styles: [path.join(__dirname, "src/main/scss/styles.scss")],
   },
   output: {
-    path: path.join(__dirname, "src/main/webapp/jsbundles"),
+    path: path.join(__dirname, "war/src/main/webapp/jsbundles"),
```

Comment:
This is still messy, but no worse than it was before this PR. Ideally they would be produced in a `target/` directory (ideally in the root project, where the sources now live) and fetched from there in the WAR module's configuration, but that is a project for another day.